quickevent/app/plugins/Competitors/src/competitorwidget.cpp

```diff
@@ -184,6 +184,15 @@ CompetitorWidget::CompetitorWidget(QWidget *parent) :
 			});
 		}
 	}
+
+	// if there is only one run propagate windget SI card change from competitors to runs
+	connect(ui->spinBox, qOverload<int>(&QSpinBox::valueChanged),[=](int newSiNumber) // spinbox= widget SI card edit box
```

Comment:
* please rename `spinBox` to `edSiId` in form designer
* rename `newSiNumber` to `new_si_number`, code style :)  https://github.com/fvacek/quickbox/wiki/Code-style

Reply:
spinbox to máš pojmenované ty, ostatní položky edit okna jsou pojmenované label 1-6. :) 
Ale můžu to všude přepsat. ;)